pkg/download/jsoncreator/json_creator.go

```diff
@@ -131,6 +132,33 @@ func replaceKeyProperties(dat map[string]interface{}) map[string]interface{} {
 	return dat
 }
 
+// removes key with specified path
+func removeKey(dat map[string]interface{}, key []string) map[string]interface{} {
+	if len(key) == 0 {
+		return dat
+	} else if len(key) == 1 {
+		delete(dat, key[0])
+		return dat
+	} else {
+		if dat[key[0]] == nil { // no field: nothing to do
+			return dat
+		}
+		if field, ok := dat[key[0]].(map[string]interface{}); ok {
+			dat[key[0]] = removeKey(field, key[1:])
+		} else if arrayOfFields, ok := dat[key[0]].([]interface{}); ok {
+			for i := range arrayOfFields {
+				if field, ok := arrayOfFields[i].(map[string]interface{}); ok {
+					arrayOfFields[i] = removeKey(field, key[1:])
+				}
+			}
+			dat[key[0]] = arrayOfFields
+		} else {
+			util.Log.Error("unexpected type")
```

Comment:
```suggestion
			util.Log.Error("Unable to parse configuration to remove identifier property %v. Correct configuration manually if necessary.", key[0])
```

Or something similar would be more helpful to a user encountering this error :)